_includes/pbc/all/install-features/202304.0/install-warehouse-user-management-feature.md

```diff
@@ -0,0 +1,319 @@
+|------------------------------------------------------|--------------------------------------------------------------|---------------|-------------------------------------------------------------|
+| WarehouseUserLoginRestrictionPlugin                  | Restricts access to the Back office for warehouse users.         |               | Spryker\Zed\WarehouseUser\Communication\Plugin\SecurityGui  |
+| WarehouseUserAssignmentUserTableActionExpanderPlugin | Expands the **User** table with the **Assign Warehouses** button.    |               | Spryker\Zed\WarehouseUserGui\Communication\Plugin\User      |
+| WarehouseUserAssignmentUserFormExpanderPlugin        | Expands the User form with the `is_warehouse_user` checkbox. |               | Spryker\Zed\WarehouseUserGui\Communication\Plugin\User      |
+| WarehouseUserAssignmentsResourcePlugin               | Registers the `warehouse-user-assignments` resource.         |               | Spryker\Glue\WarehouseUserBackendApi\Plugin\GlueApplication |
+
+**src/Pyz/Zed/SecurityGui/SecurityGuiDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\SecurityGui;
+
+use Spryker\Zed\SecurityGui\SecurityGuiDependencyProvider as SprykerSecurityGuiDependencyProvider;
+use Spryker\Zed\WarehouseUser\Communication\Plugin\SecurityGui\WarehouseUserLoginRestrictionPlugin;
+
+class SecurityGuiDependencyProvider extends SprykerSecurityGuiDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\SecurityGuiExtension\Dependency\Plugin\UserLoginRestrictionPluginInterface>
+     */
+    protected function getUserLoginRestrictionPlugins(): array
+    {
+        return [
+            new WarehouseUserLoginRestrictionPlugin(),
+        ];
+    }
+}
+
+```
+
+**src/Pyz/Zed/User/UserDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\User;
+
+use Spryker\Zed\User\UserDependencyProvider as SprykerUserDependencyProvider;
+use Spryker\Zed\WarehouseUserGui\Communication\Plugin\User\WarehouseUserAssignmentUserFormExpanderPlugin;
+use Spryker\Zed\WarehouseUserGui\Communication\Plugin\User\WarehouseUserAssignmentUserTableActionExpanderPlugin;
+
+class UserDependencyProvider extends SprykerUserDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\UserExtension\Dependency\Plugin\UserTableActionExpanderPluginInterface>
+     */
+    protected function getUserTableActionExpanderPlugins(): array
+    {
+        return [
+            new WarehouseUserAssignmentUserTableActionExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\UserExtension\Dependency\Plugin\UserFormExpanderPluginInterface>
+     */
+    protected function getUserFormExpanderPlugins(): array
+    {
+        return [
+            new WarehouseUserAssignmentUserFormExpanderPlugin(),
+        ];
+    }
+}
+
+```
+
+---
+{% info_block warningBox "Verification" %}
```

Comment:
I've rewritten the Verification steps, so review them and make sure they are correct since I don't have this feature to be able to test it in the Back Office on my own.

Reply:
Also looks good 👍